internal/pac.go

```diff
@@ -7,20 +7,28 @@ import (
 	"net/url"
 	"strings"
 	"sync"
+	"io/ioutil"
 
 	"github.com/jackwakefield/gopac"
 	log "github.com/sirupsen/logrus"
 )
 
 type PAC struct {
-	File   string
+	File    string
+	Source  []byte
 	Parser *gopac.Parser
 	mux    sync.Mutex
 }
 
 func (pac *PAC) Init() error {
 	pac.Parser = new(gopac.Parser)
-	return pac.Parser.Parse(pac.File)
+	src, err := ioutil.ReadFile(pac.File)
+	if err != nil {
```

Comment:
Checking for `err != nil` is correct according to the docs: https://pkg.go.dev/io/ioutil#ReadFile